inspirehep/records/indexer/tasks.py

```diff
@@ -69,29 +69,31 @@ def index_record(self, uuid, record_version=None, force_delete=None):
         list(dict): Statistics from processing references.
 
     """
-    logger.info(
-        f"Starting shared task `index_record` for record {uuid}:v{record_version}"
+    LOGGER.info(
+        "Starting shared task `index_record` for record {%r}:v{%s}",
```

Comment:
if we rename the function, this will be wrong :P what about `__name__` instead of `index_record` hardcoded?

Reply:
bu `__name__` returns module name, not function name. Message saying:

> Starting shared task `inspirehep.records.indexer.tasks` for record ...
won't be understandable at all, and it's easier to search through the code for hardcoded message.